package/MDAnalysis/analysis/hbonds/hbond_analysis.py

```diff
@@ -1030,11 +1031,12 @@ def _get_timestep():
     @staticmethod
     def calc_angle(d, h, a):
```

Comment:
To make this handle PBC you need to add `box=None` as an optional kwarg to the signature here, then pass that to `calc_angles` (passing `None` to calc_angles is ok).  It's only possible to do PBC with the box information